src/site/_includes/header.html

```diff
@@ -77,7 +77,7 @@
 
     <div class="navbar navbar-fixed-top navbar-inverse" role="navigation">
       <div class="dart-developer-summit-reg" style="text-align:center;background-color: hsl(172,67%,80%);">
-        <a href="/events/2015/summit/">Watch sessions from the first Dart Developer Summit</a>
+        <a href="https://www.youtube.com/playlist?list=PLOU2XLYxmsIIQorIS8gagUiMau9S84vZV">Watch sessions from the first Dart Developer Summit</a>
```

Comment:
Should we open this in a new window? Or at least attach an event to this so we can count it specially, somehow? It feels weird to have such a prominent link sending people away from dartlang.org.